### DISCUSSION TITLE
Shell history and Warnings

### DISCUSSION THREAD

**Discussion by hakavlad** [samuel-lucas6/Kryptor#80] [Ideas]
https://www.kryptor.co.uk/tutorial/entering-a-passphrase#non-interactively

```sh
$ kryptor -e -p:"YeV$Ra7{_IJ]nN2v1m,ei9d+T" file.txt
```

On Linux and macOS, the history of entered commands is saved not only for the current session but also between reboots by default.

The problem concerns not only passwords but also file paths.

There is no information about the problem in the Warnings.

Hint: On Linux and macOS (for `bash` and `zsh`), saving in shell history can be prevented by adding a space before the command.

https://www.kryptor.co.uk/security-limitations

It may also be worth updating this section to add a warning that `Kryptor` itself cannot prevent the saving of the command history; this responsibility lies with the user.

**Comment by samuel-lucas6** [comment]
Sorry for not responding to this. I somehow missed it. This is worth a mention; I'll try to update the tutorial at some point.
